elisp/intero.el

```diff
@@ -1807,7 +1807,7 @@ suggestions are available."
                                       "’ with ‘"
                                       (plist-get suggestion :replacement)
                                       "’")
-                       :default nil))))
+                       :default (null (cdr intero-suggestions))))))
```

Comment:
Technically you can get more than one `Not in scope` error at once. But I'll follow that up with a patch later.